apps/build.info

```diff
@@ -9,7 +9,7 @@ IF[{- !$disabled{apps} -}]
           genpkey.c genrsa.c nseq.c ocsp.c passwd.c pkcs12.c pkcs7.c pkcs8.c \
           pkey.c pkeyparam.c pkeyutl.c prime.c rand.c req.c rsa.c rsautl.c \
           s_client.c s_server.c s_time.c sess_id.c smime.c speed.c spkac.c \
-          srp.c ts.c verify.c version.c x509.c rehash.c \
+          srp.c ts.c verify.c version.c x509.c rehash.c storeutl.c \
```

Comment:
Maybe...  I was thinking that if the application expands to do more than just listing and fetching, should everything be under the same sub-command?  I'm for example thinking of `rsa` and `rsautl`, not to mention `genrsa`...  maybe I'm overthinking it...